client/state/selectors/has-user-purchased-a-plan.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * External dependencies
+ *
+ * @format
+ */
+import { some } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import createSelector from 'lib/create-selector';
+import { isPlan } from 'lib/products-values';
+
+/**
+ * Get the newest site of the current user
```

Comment:
Wrong docblock

Reply:
Also fixed in https://github.com/Automattic/wp-calypso/pull/19421/commits/f08d29e1f7ea26202d16bdfc0d003b8365b92b80.